tailwind.base.config.js

```diff
@@ -69,6 +69,7 @@ module.exports = {
       },
       boxShadow: {
         'xl-hover': '0 20px 25px -5px rgb(0 0 0 / 0.2)',
+        navbar: '0px 0px 5.1px 0px var(--color-primary-darkest)',
```

Comment:
This is not appropriate I think; why not specify the box shadow in the HTML?

Reply:
Ok, wasn't sure this was possible. What's the syntax for this?

Reply:
not sure but if needed, you can just use CSS instead of tailwind classes

Reply:
Hum, ok I thought it would be preferable to have a custom tailwind class than CSS in our components. Does that mean that in the long run we should get rid of all the custom classes in the tailwind config?

Reply:
In my opinion yes, global classes are technical debt unless they adress standard components like cards, buttons, icons, inputs...

Reply:
Using tailwind classes or css in the component is honestly almost the same, sure we try to have as few CSS as possible in the project but it's preferable to have local CSS than global classes

Reply:
Ok I used tailwind interpolation